mmv1/third_party/terraform/services/dataproc/resource_dataproc_cluster.go

```diff
@@ -1227,6 +1228,35 @@ func ResourceDataprocCluster() *schema.Resource {
 														},
 													},
 												},
+												"provisioning_model_mix": {
+													Type:     schema.TypeList,
+													Optional: true,
+													Computed: true,
```

Comment:
I don't think `Computed` is needed here. It is used for the case that a default value is returned from API side when no value for this field is provided in the request.

Reply:
Updated. Removed the `Computed`